README.md

```diff
@@ -198,3 +198,5 @@ If you find any issue while using those, please open an issue or a pull request.
 Please refer to Habana Gaudi's official [installation guide](https://docs.habana.ai/en/latest/Installation_Guide/index.html).
 
 > Tests should be run in a Docker container based on Habana Docker images.
+>
+> This repository has been validated for SynapseAI 1.6.
```

Comment:
Should we say "The current version has been validated for SynapseAI 1.6"

Reply:
Sounds good. I'm thinking also about specifying it for both the source code of the Github repo and the latest release. Something like:
> The source code in this repository and the latest release 1.2.2 have been validated for SynapseAI 1.6.

What do you think @libinta ?